lib/src/components/stack/Ancestor_Stack.res

```diff
@@ -1 +1,698 @@
+module Make = (Maker: Ancestor_StylesMaker.T) => {
+  module Base = Ancestor_Base.Make(Maker)
+  module Styles = Ancestor_Styles.Make(Maker)
 
+  %%private(
+    let defaultStyles = Maker.css(`
```

Comment:
This `defaultStyles` class is only applied when there is no value defined to the `spacing` prop. I think we can keep the display flex, flex wrap, etc but should always apply as a default className for the stack component. I mean, you can rename to stackStyles or something like that and always pass a className to the Base component. 
Check the line 374, I would do something like:
```rescript
className={`${stackStyles} ${stack(~spacing?, ())} ${className}`}
```